ibmcloud/terraform/cluster/ansible/kube-playbook.yml

```diff
@@ -0,0 +1,270 @@
+#
+# (C) Copyright IBM Corp. 2022.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+- hosts: all
+  remote_user: root 
+  tasks:
+    - name: Add Kubernetes apt repository signing key
+      apt_key:
+        url: https://packages.cloud.google.com/apt/doc/apt-key.gpg
+
+    - name: Add Kubernetes apt repository
+      apt_repository:
+        repo: deb https://apt.kubernetes.io/ kubernetes-xenial main
+        filename: kubernetes
+
+    - name: Install deb packages
+      apt:
+        name:
+          - "linux-modules-extra-{{ ansible_kernel }}"
+          - build-essential
+          - jq 
+          - libseccomp-dev
+          - pkg-config
+          - kubelet
+          - kubeadm
+          - kubectl
+          - kubernetes-cni
+          - cri-tools
+          - git 
+          - kpartx
+
+    - name: Install deb packages for Ubuntu 20.04 or later 
+      apt:
+        name:
+          - libbtrfs-dev
+      when:
+        - ansible_facts['distribution'] == "Ubuntu"
+        - ansible_facts['distribution_major_version'] | int >= 20
+
+    - name: Install deb packages for Ubuntu 18.04 or before
+      apt:
+        name:
+          - btrfs-tools
+      when:
+        - ansible_facts['distribution'] == "Ubuntu"
+        - ansible_facts['distribution_major_version'] | int < 20
+
+    - name: Hold deb packages 
+      dpkg_selections:
+        name: "{{ item }}"
+        selection: hold
+      with_items:
+        - kubelet
+        - kubeadm
+        - kubectl
+        - kubernetes-cni
+        - cri-tools
+
+    - name: Add local IP address in /etc/hosts
+      shell: |
+        if ! grep -q '^[^#].*\b{{ ansible_hostname }}\b' /etc/hosts; then
+          echo '{{ ansible_default_ipv4.address }} {{ ansible_hostname }}' >> /etc/hosts || exit 2
+          exit 1
+        fi
+      register: result
+      changed_when: result.rc == 1
+      failed_when: result.rc > 1
+
+    - name: Disable swap
+      shell: |
+        [ -z "$(swapon --show)" ] && exit 0
+        swapoff --all && exit 1
+        exit 2
+      register: result
+      changed_when: result.rc == 1
+      failed_when: result.rc > 1
+
+    - name: Disable swap in fstab
+      replace:
+        path: /etc/fstab
+        regexp: '^([^#\s]+\s+){2}swap\s'
+        replace: '# \1'
+
+    - name: Configure kernel modules to load at boot
+      copy:
+        dest: "{{ item.path }}"
+        content: "{{ item.content }}"
+      with_items:
+        - path: /etc/modules-load.d/k8s.conf
+          content: br_netfilter
+        - path: /etc/modules-load.d/peerpod.conf
+          content: vrf
+
+    - name: Load kernel modules
+      modprobe:
+        name: "{{ item }}"
+      with_items:
+        - br_netfilter
```

Comment:
We should test the playbooks work if the Configure kernel modules to load at boot and load kernel modules task in the kubernetes playbook only sets up the br_netfilter kernel module.

Reply:
Tested and successfully reran Terraform template.
Remove vrf from Kube-playbook, and br_netfilter from Kata-playbook under Load kernel modules . Also, remove vrf configuration from Kube-playbook, and br_netfilter configuration from Kata-playbook under Configure kernel modules to load at boot.

Reply:
I've tested the new playbooks and they work with enabling the br_netfilter kernel module only being in the kube playbook.